Cargo.toml

```
@@ -61,3 +61,6 @@ v1_22 = []
 [package.metadata.docs.rs]
 # docs.rs generates docs for the latest version. To see the docs for an older version, please generate them yourself.
 features = ["v1_22"]
+
+[workspace]
+members = ["k8s-openapi-codegen", "k8s-openapi-codegen-common", "k8s-openapi-derive", "k8s-openapi-tests", "k8s-openapi-tests-macro-deps"]
```

```suggestion
members = [
	"k8s-openapi-codegen",
	"k8s-openapi-codegen-common",
	"k8s-openapi-derive",
	"k8s-openapi-tests",
	"k8s-openapi-tests-macro-deps",
]
```

ie split over multiple lines and with a newline at the end of the file.